platform/android/config.cmake

```diff
@@ -250,11 +250,6 @@ target_link_libraries(mapbox-gl
     PUBLIC -Wl,--gc-sections
 )
 
-# Create a stripped version of the library and copy it to the JNIDIR.
```

Comment:
Does this still happen somewhere else? Do we need to update [metrics.sh](https://github.com/mapbox/mapbox-gl-native/blob/36a04e49c834f52828dd26871641517c000c45f5/platform/android/scripts/metrics.sh#L7-L12)?